app/frontend/src/features/constants.ts

```diff
@@ -101,6 +103,23 @@ export const WHEELCHAIR = "Wheelchair accessible";
 // Edit home
 export const EDIT_HOME = "Edit home";
 
+// References
+export const REFERENCES_FILTER_A11Y_LABEL = "Show references: ";
+export const referencesFilterLabels = {
+  [ReferenceType.REFERENCE_TYPE_FRIEND]: "From friends",
+  [ReferenceType.REFERENCE_TYPE_HOSTED]: "From guests",
+  [ReferenceType.REFERENCE_TYPE_SURFED]: "From hosts",
+  all: "All references",
+  given: "Given to others",
+};
+export const referenceBadgeLabel = {
+  [ReferenceType.REFERENCE_TYPE_FRIEND]: "Friend",
+  [ReferenceType.REFERENCE_TYPE_HOSTED]: "Guest",
+  [ReferenceType.REFERENCE_TYPE_SURFED]: "Hosted",
+};
+export const NO_REFERENCES = "No references of this kind yet!";
+export const getReferencesGivenHeading = (name: string) =>
+  `References ${name} wrote`;
```

Comment:
The merge probably just merged the old stuff with this branch, but this is not needed as I've moved these into `features/profile/constants.ts`